cockatrice/src/pictureloader.cpp

```diff
@@ -384,11 +384,9 @@ void PictureLoaderWorker::startNextPicDownload()
         picDownloadFailed();
     } else {
         QUrl url(picUrl);
-        qDebug() << qPrintable(QString("PictureLoader: [card: %1 set: %2]: Trying to %3: %4")
-                                   .arg(cardBeingDownloaded.getCard()->getCorrectedName())
-                                   .arg(cardBeingDownloaded.getSetName())
-                                   .arg(picDownload ? "download picture from url" : "load picture from cache")
-                                   .arg(url.toDisplayString()));
+        qDebug() << "PictureLoader: [card: " << cardBeingDownloaded.getCard()->getCorrectedName()
+                 << " set: " << cardBeingDownloaded.getSetName()
```

Comment:
`]: Trying to ` seems to have disappeared?

Reply:
Indeed, put it back in